frontend/src/app/uploads/DeviceLookup/DeviceLookup.test.tsx

```diff
@@ -64,6 +79,9 @@ describe("Device lookup", () => {
 
     expect(screen.getByText("Celoxitin")).toBeInTheDocument();
     expect(screen.getByText("Model A")).toBeInTheDocument();
+
+    // `getByText` throws if multiple matches are found - this device has two
+    // SupportedDiseaseTestPerformed entries, implying the de-duplication is occurring
     expect(screen.getByText("COVID-19")).toBeInTheDocument();
```

Comment:
I recommend changing this to getAllByText and assert on `size = 1` if that's the intended assertion

Reply:
Hah, I did that at first! The funny thing is that ESLint thinks that `expect(screen.getAllByText("foo")).toHaveLength(1)` a clumsy attempt at checking for the mere existence of text and "helpfully" re-writes it to what we see here 😅 

<img width="851" alt="Screen Shot 2023-05-18 at 2 51 58 PM" src="https://github.com/CDCgov/prime-simplereport/assets/27730981/3c2d8874-41f8-4a13-9983-0f741e92440b">

I was just going to capitulate to the linter, but I think you're right and it would be clearer to just slap an `eslint-disable` on those lines ([b11f2d4](https://github.com/CDCgov/prime-simplereport/pull/5834/commits/b11f2d41839ab18d772c09b3b73045619499766b)).